scripts/bootstrap-aio.yml

```diff
@@ -151,12 +151,15 @@
     rpco_deploy_hardening: "{{ lookup('env', 'DEPLOY_HARDENING') }}"
     TRIGGER: "{{ lookup('env', 'TRIGGER')}}"
     SCENARIO: "{{ lookup('env', 'SCENARIO')}}"
-    TARGET: "{{ lookup('env', 'TARGET')}}"
+    TARGET: "{{ target_override | default(lookup('env', 'TARGET')) }}"
```

Comment:
`lookup('env', 'TARGET')|default('aio')` then `set env.TARGET=mnaio` in multi_node_aio.yml, similar to https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/rpc_aio.yml#L290 ? Then in jobs that don't set target, aio will be used by default (eg the artifact jobs).